src/test/mir-opt/early_otherwise_branch.opt1.EarlyOtherwiseBranch.diff

```diff
@@ -52,13 +52,13 @@
 -     }
 - 
 -     bb3: {
-          StorageLive(_8);                 // scope 0 at $DIR/early_otherwise_branch.rs:5:15: 5:16
-          _8 = (((_3.0: std::option::Option<u32>) as Some).0: u32); // scope 0 at $DIR/early_otherwise_branch.rs:5:15: 5:16
```

Comment:
These tuple patterns swapped their order, that seems wrong (wrt drop order and such), though I can't tell how it's happening, did you bless the mir-opt tests?

Reply:
yup. i blessed mir-opt.